LayoutTests/imported/w3c/web-platform-tests/svg/animations/scripted/animatetransform-type-missing-value-default-expected.txt

```diff
@@ -0,0 +1,6 @@
+
+PASS <animateTransform> 'type' attribute missing/invalid value default, "type" attribute is "translate"
+PASS <animateTransform> 'type' attribute missing/invalid value default, missing "type" attribute
+PASS <animateTransform> 'type' attribute missing/invalid value default, invalid "type" attribute
+FAIL <animateTransform> 'type' attribute missing/invalid value default, removed "type" attribute assert_equals: expected 70 but got 0
```

Comment:
What's the deal with this failure?

Reply:
I have separate [bug](https://bugs.webkit.org/show_bug.cgi?id=257906) for it. I tried various things locally and trying to even do 'Blink' merge, which is not easy and even tried to wrap in `if (oldValue != newValue)` but didn't had much success.

Reply:
@nikolaszimmermann - It passes, if I modify:
`m_type = SVGTransformable::parseTransformType(newValue).value_or(SVGTransformValue::SVG_TRANSFORM_UNKNOWN);`

Reply:
I would say that we should land this and track other failing test in separate patch / bug since it is quick win.